<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataDisplaySettings</class>
 <widget class="QFrame" name="DataDisplaySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>201</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <widget class="QTabWidget" name="m_panel_settings">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>741</width>
     <height>191</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="m_tab_settings">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <attribute name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/slide_up.svg</normaloff>:/images/slide_up.svg</iconset>
    </attribute>
    <attribute name="title">
     <string>Display Settings</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <item row="1" column="0">
      <layout class="QGridLayout" name="m_grid_settings">
       <item row="1" column="3">
        <widget class="QLabel" name="m_lb_font_colour">
         <property name="text">
          <string>Font Colour</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="m_lb_text_size">
         <property name="text">
          <string>Text size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="m_lb_bg_colour">
         <property name="text">
          <string>Background Colour</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="m_sp_fontsize"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="m_lb_textType">
         <property name="text">
          <string>Text Type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="m_bt_bg_colour">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 240)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="m_bt_font_colour">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="m_lb_font">
         <property name="text">
          <string>Font</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="m_cb_textType"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QFontComboBox" name="m_cb_fonts"/>
       </item>
      </layout>
     </item>
     <item row="1" column="3">
      <spacer name="h_spacer_panel">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="DataDisplay" name="m_preview_display"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="m_display_control_bar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>36</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="m_h_controls_wrapper">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="m_bt_clear">
      <property name="text">
       <string>&amp;Clear</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_check_hex_out">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Show the output in hexadecimal format</string>
      </property>
      <property name="text">
       <string>He&amp;x output</string>
      </property>
      <property name="shortcut">
       <string>Alt+X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_check_timestamp">
      <property name="text">
       <string>Show &amp;Timestamp</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="m_check_lineBreak">
      <property name="text">
       <string>Display &amp;Ctrl characters</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="h_spacer_control_bar">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataDisplay</class>
   <extends>QFrame</extends>
   <header>datadisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_panel_settings</tabstop>
  <tabstop>m_check_hex_out</tabstop>
  <tabstop>m_check_timestamp</tabstop>
  <tabstop>m_check_lineBreak</tabstop>
  <tabstop>m_cb_fonts</tabstop>
  <tabstop>m_sp_fontsize</tabstop>
  <tabstop>m_bt_font_colour</tabstop>
  <tabstop>m_bt_bg_colour</tabstop>
  <tabstop>m_bt_clear</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
